src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
+
+estimate.py
+
+Functions to help with estimation of low-light parameters.
+
+"""
+
+import sys
+sys.path.insert(0, '../../sipmpdf')
+sys.path.insert(0, '../')
+
+import matplotlib.pyplot as plt
+import src.sipmanalyze.formats as forms
```

Comment:
Following the installation instructions, you should be able to use `import sipmanalyze.formts` Let me know if that doesn't work out for you.